### DISCUSSION TITLE
Encrypt specific fields in mysql DB

### DISCUSSION THREAD

**Discussion by sweepkim** [grafana/grafana#64272] [General]
I am using mysql as Grafana DB and I found only `password` column is encrypted.

If a user specifies a column(e.g. `login`, `email`, `name`) to be encrypted in Grafana.ini , 
Are there any plans to support the ability to encrypt those columns as well?

This would be very helpful in securing User's email information.

thanks in advance.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). We are migrating this discussion to an issue and closing the discussion. The issue is #82801. Feel free to continue the discussion around this there. Thank you!
